client/src/components/JurisdictionAdmin/BatchRoundDataEntry.tsx

```diff
@@ -129,15 +129,20 @@ const BatchResultsForm = ({
         : undefined,
   })
 
-  const onSubmit = (results: IBatchResults) => {
+  const onSubmit = async (results: IBatchResults) => {
     const tallySheet = {
       name: 'Tally Sheet #1',
       results,
     }
-    recordBatchResults.mutate(
-      { batchId: batch.id, resultTallySheets: [tallySheet] },
-      { onSuccess: closeForm }
-    )
+    try {
+      await recordBatchResults.mutateAsync({
+        batchId: batch.id,
+        resultTallySheets: [tallySheet],
+      })
+    } catch (e) {
+      // Do nothing - errors toasted by queryClient
+    }
```

Comment:
not a huge fan of this empty catch pattern but I don't have any better ideas

Reply:
Out of curiosity, what would happen if we just didn't wrap in a try-catch at all? It sounds like we expect `.mutateAsync` to handle all errors internally

Reply:
`mutateAsync` will return a rejected promise (which throws an exception when `await`ed), which won't be caught and will crash the app.

I don't love having the empty try-catch, but haven't been able to think of a better solution. Open to ideas